### DISCUSSION TITLE
Release v1.3.3

### DISCUSSION THREAD

**Discussion by gelmio** [onesine/react-tailwindcss-datepicker#33] [Q&A] [answered]
Love the library, great work 👍 
Any chance we can get the latest release (v1.3.3) released on NPM? It's got a bug fix that I needed.
Thanks

**Comment by onesine** [comment, marked as answer]
Sorry for the delay @gelmio . I just published it. You can enjoy it.


Thanks for using this library.

**Reply by gelmio** [reply]
Thanks @onesine!
I've run an `npm install` and I'm running into issues with the new package. Vite doesn't seem to think that it exists. When I import Datepicker into my component (`import Datepicker from "react-tailwindcss-datepicker";`) I get the error message: `Cannot find module 'react-tailwindcss-datepicker' or its corresponding type declarations.`

And then when running Vite I get `[ERROR] [plugin vite:dep-scan] Failed to resolve entry for package "react-tailwindcss-datepicker". The package may have incorrect main/module/exports specified in its package.json.`

Are you having this problem at your end?

**Reply by onesine** [reply]
Thank you for your feedback.

Please install the specific version 1.3.2 until we fix this problem. 

Thank you for using our package.

**Reply by gelmio** [reply]
Will do. Thanks again for this package, really appreciate it.

If it's helpful, here's the file structure of the installed package in my node_modules
![image](https://user-images.githubusercontent.com/23208546/211458335-24e5439c-0caf-45f2-ba36-9b5df58ab835.png)

**Reply by onesine** [reply]
Thank you very much for this information
